installer/Migration/V5_0_0/translation/bg_BG.yaml

```diff
@@ -452,3 +452,12 @@ translations:
     - { target: 'Служител може да редактира / изтривате собствените записи посещаемост', unitId: employee_can_edit_delete_own_attendance_records, group: attendance }
     - { target: 'Надзорен орган могат да добавят / редактиране / изтриване на записи на посещаемост на подчинените', unitId: supervisor_can_add_edit_delete_attendance_records_of_subordinates, group: attendance }
     - { target: 'Присъствието ми записи', unitId: my_attendance_records, group: attendance }
+    - { target: 'Добавен успешно', unitId: successfully_deleted, group: general }
```

Comment:
Manually added these strings so hereafter we cannot use the current yml generator. Some of these weren't detected to insignificant changes such as case or even the punctuation mark